lib/pipeline.js

```diff
@@ -639,6 +640,11 @@ class PipelineModel extends BaseModel {
             })
             .then(p => {
                 logger.info(`pipelineId:${this.id}: Creating child pipeline for ${scmUrl} with pipelineId:${p.id}.`);
+                // sync pipeline to create jobs
+                p.sync();
+                if (SD_API_URI) {
+                    p.addWebhook(`${SD_API_URI}/v4/webhooks`);
```

Comment:
same here
does it also need `await`?

Reply:
were we not adding webhooks before ? I wonder how it's working in our setup 🤔

Reply:
Currently, child pipelines are only created or updated when the parent pipeline is synced.
https://github.com/screwdriver-cd/models/blob/56e3ce9b02c68b99ca5848f3e9e029702f742f11/lib/pipeline.js#L730
However, child pipeline's webhooks are not added in this method.
The parent pipeline's webhook is added after it is created, but the child pipeline's ones are not.
https://github.com/screwdriver-cd/screwdriver/blob/cd5c538ba159f0229b7307f491eb4000892a47f4/plugins/pipelines/create.js#L120
So we should add webhooks after child pipeline is created.